src/EditorFeatures/VisualBasicTest/QualifyMemberAccess/QualifyMemberAccessTests.vb

```diff
@@ -147,6 +169,19 @@ CodeStyleOptions.QualifyFieldAccess)
 CodeStyleOptions.QualifyFieldAccess)
         End Function
 
+        <WorkItem(28509, "https://github.com/dotnet/roslyn/issues/28509")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)>
+        Public Async Function QualifyFieldAccess_NotSuggestedOnLocalVarInObjectInitializer() As Task
```

Comment:
❕ This is a collection initializer, not an object initializer (test is correct, but the method name here is wrong)